### DISCUSSION TITLE
Saved Tutorial Files -- Octave

### DISCUSSION THREAD

**Discussion by lhragan** [thliebig/openEMS-Project#83] [General]
Thorsten, I noticed that you confessed to suffering a catastrophic server failure.  For all the work you had invested, I am sorry it happened.  I too suffered a failure of my workstation, and fumbled the recovery with a new one.  
I did, however, recover the tutorial files you mentioned needing.  They are attached in the tar.gz file.  
Also included is a modified version of the helical antenna script -- I included the constants, along with a reference.  It runs under the latest installation on my linuxmint (Ubuntu 20.+) machine.
I used that file to troubleshoot my latest installation, locating a previous version of the library file that was not overwritten in the new install.  I am almost back in business -- only need to sort out the various file formats for paraview ...
BTW, I am really grateful for the multi-processor mod.  I run some rather complex structures in an attempt to model a physical system, and the faster executing times are much appreciated.
Thank you sir,
[Tutorials.tar.gz](https://github.com/thliebig/openEMS-Project/files/11716280/Tutorials.tar.gz)

**Comment by thliebig** [comment]
Well it was not me suffering that failure, but rather the university hosting the site at the time. 
In any case, thanks for the tutorials, I will have a look if anything missing is includes. From a first look it seems that most (or all?) of them always have been on github though?

**Comment by lhragan** [comment]
It was several years ago that I downloaded these.  2016 or 2017?  They were
all that were available at the time.
While it was the university host that failed, you and the other
contributors (along with all of us users) were among those who suffered.
In any case, I appreciate all the effort.  FYI, I am a retired radio
frequency engineer.  83 years old...
Lawrence Ragan
Austin, TX
